### DISCUSSION TITLE
Anyway to use this on unraid?

### DISCUSSION THREAD

**Discussion by aasquith94** [mrlt8/docker-wyze-bridge#51] [Q&A]
as title states. trying to run on unraid but their dockers work differently. wondering if anyone has had any success

**Comment by mrlt8** [comment]
I believe [another user](https://github.com/mrlt8/docker-wyze-bridge/issues/7) was able to get it running in the past

**Reply by mckochan** [reply]
Yes, really simple following the video guide https://www.youtube.com/watch?v=yD6FJEYRpsY. You basically make a new folder in appdata then follow directions in readme (do a git pull, copy config file, nano edit it, docker-compose up -d). There is a command from the video guide you need to run to install docker compose. 
And since version 0.5, there is only one container created, not two.

**Comment by mrlt8** [comment]
Would something like this [xml](https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-wyze-bridge.xml) work?

I don't have unraid so can't test it out. 
Would appreciate if someone with unraid experience could help out!

**Reply by aasquith94** [reply]
`</Containers>` at the end needs to be `</Container>`. Then it imports great into unraid. I'm having a little trouble getting the streams to work but the log looks all correct so I need to determine if its the xml or my network before I can comment further on its success.

**Reply by mitchross** [reply]
You are close. It will look like

https://gist.github.com/mitchross/a108bde2fd6a621202b9cb80a57b390c

I just tested this and it works fine.

It would be great to add to the official unraid community apps. You are almost there, just need to follow https://forums.unraid.net/topic/57181-docker-faq/#comment-566084
